### PR TITLE
owl-ode, owl-ode-sundials: release 0.1.0

### PR DESCRIPTION
**Pull request by mseri**
This pull-request concerns:
-`owl-ode.0.1.0`: Owl's ODE solvers
-`owl-ode-sundials.0.1.0`: Owl's ODE solvers, interface with SundialsML

---
* Homepage: https://github.com/owlbarn/owl_ode
* Source repo: git+https://github.com/owlbarn/owl_ode.git
* Bug tracker: https://github.com/owlbarn/owl_ode/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by mseri** [comment]
Unfortunately in debian buster libsundials-serial-dev is no longer present and has been replaced by libsundials-dev. So the build will always fail

**Comment by mseri** [comment]
The 4.08.0 failure happens during the tests due to the usual mysterious error:
```
          - owl-ode → owl >= 0.6.0 → eigen >= 0.1.0 → ctypes >= 0.14.0 → lwt < 4.0.0 → ocaml < 4.08.0
```

File: packages/owl-ode/owl-ode.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+maintainer: "owlbarn"
+authors: [ "Marcello Seri" "Ta-Chu Calvin Kao" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/owl_ode"
+dev-repo: "git+https://github.com/owlbarn/owl_ode.git"
+bug-reports: "https://github.com/owlbarn/owl_ode/issues"
+doc: "https://owlbarn.github.io/owl_ode/owl-ode"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Comment (mseri) [inline packages/owl-ode/owl-ode.0.1.0/opam]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test & ocaml:version < "4.08.0"}
```

File: packages/owl-ode-sundials/owl-ode-sundials.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+maintainer: "owlbarn"
+authors: [ "Marcello Seri" "Ta-Chu Calvin Kao" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/owl_ode"
+dev-repo: "git+https://github.com/owlbarn/owl_ode.git"
+bug-reports: "https://github.com/owlbarn/owl_ode/issues"
+doc: "https://owlbarn.github.io/owl_ode/ode"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "exec" "examples/van_der_pol.exe"] {with-test}
```
Comment (mseri) [inline packages/owl-ode-sundials/owl-ode-sundials.0.1.0/opam]:
```suggestion
  ["dune" "exec" "examples/van_der_pol.exe"] {with-test & ocaml:version < "4.08.0"}
```

**Comment by kit-ty-kate** [comment]
> Unfortunately in debian buster libsundials-serial-dev is no longer present and has been replaced by libsundials-dev. So the build will always fail

Doesn't it make more sense to change it since the old one is not supposed to be used anymore?

**Comment by mseri** [comment]
I've modified it, but the ocaml sundials version targets a very specific version, so I don't know if it is going to be working or not. If we see that it does not work, I'll remove the commit

File: packages/owl-ode-sundials/owl-ode-sundials.0.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+maintainer: "owlbarn"
+authors: [ "Marcello Seri" "Ta-Chu Calvin Kao" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/owl_ode"
+dev-repo: "git+https://github.com/owlbarn/owl_ode.git"
+bug-reports: "https://github.com/owlbarn/owl_ode/issues"
+doc: "https://owlbarn.github.io/owl_ode/ode"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "exec" "examples/van_der_pol.exe"] {with-test & ocaml:version < "4.08.0"}
```
Comment (mseri) [inline packages/owl-ode-sundials/owl-ode-sundials.0.1.0/opam]:
```suggestion
```

File: packages/owl-ode-sundials/owl-ode-sundials.0.1.0/opam [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+]
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "owl" {>= "0.6.0"}
+  "dune" {>= "1.5.0"}
+  "owl-ode" {>="0.1.0"}
+  "owl-plplot" {with-test}
```
Comment (mseri) [inline packages/owl-ode-sundials/owl-ode-sundials.0.1.0/opam]:
```suggestion
```

**Comment by mseri** [comment]
Much better. I have added a workaround for the tests in docker in the next release, I am going to disable them for the moment.

**Comment by mseri** [comment]
There will be distributions not providing ploplot nor sundials (or the version we need). Those will unfortunately fail

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**